### ISSUE TITLE
Remove acAclPolicy{msiAclPolicy('STANDARD');} (strictACL as a knob)

### ISSUE THREAD

**Issue by trel** [irods/irods#7985]
- [x] irods/irods
- [x] icommands
- [x] PREP
- [x] irods_docs

---

## Feature

Deprecation is https://github.com/irods/irods/issues/6843.

This issue is about removing the `STANDARD` option.

**Comment by korydraughn** [comment]
Things to consider:
- Will the MSI continue to exist?
- How do we remove it from core.re? Do we remove it from core.re at all?
- If the MSI is removed, how does the NREP deal with the call still existing inside core.re?

**Comment by trel** [comment]
I think the msi continues to exist, but we change the implementation to be a no-op that emits a warning.   So existing rulefiles will not throw errors?   Or maybe, it emits the warning for one major release before we remove the msi completely.

**Comment by korydraughn** [comment]
So ...
- iRODS 4 prints a warning about the MSI's deprecation on server startup, not agent startup
- iRODS 5 does not provide the MSI, leading to server startup failing
  - A clear error message is logged for the admin
  - Config check could be used to verify the core.re file is good
    - i.e. Use the list rules function of the plugin

**Comment by trel** [comment]
or... if we're a little nicer...
- iRODS 4 does as you say (somewhat noisy)
- iRODS 5 complains on every agent startup (more noisy)
- iRODS 6 does not start b/c syntax error / missing MSI error

**Comment by korydraughn** [comment]
I can see it going either way, but I'm still in the camp of remove in iRODS 5 since it's a major release.

Will need to think about this a bit more.

**Comment by trel** [comment]
sure, 6 is a major release too.    so yes, either is fine i think...

removing in 5, before we've even deprecated in 4 does give me a little pause...

**Comment by korydraughn** [comment]
> removing in 5, before we've even deprecated in 4 does give me a little pause...

I agree. My comments assume the deprecation happens in 4 first. That's my bad. I should've mentioned that.

**Comment by korydraughn** [comment]
Related to #7954.

**Comment by alanking** [comment]
Need to confirm / merge https://github.com/irods/irods_rule_engine_plugin_python/pull/233, I think. Plus, take a pass over the docs.

**Comment by alanking** [comment]
I think that's everything... @MartinFlores751 - If you agree, please close. Thanks

**Comment by alanking** [comment]
Oh, and close #7954 as well
